Build in docker container not working

Hello.
I'm trying to build a skopeo binary for Linux on my Mac by running `make build`, but the build freezes to the post-install tzdata script that installs in the recommended packages. This happens due to the fact that the docker build execution is not interactive
```
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
Configuring tzdata
------------------

Please select the geographic area in which you live. Subsequent configuration
questions will narrow this down by presenting a list of cities, representing
the time zones in which they are located.

  1. Africa      4. Australia  7. Atlantic  10. Pacific  13. Etc
  2. America     5. Arctic     8. Europe    11. SystemV
  3. Antarctica  6. Asia       9. Indian    12. US
Geographic area: 6
^Cmake: *** [binary] Interrupt: 2
```

--- a/Dockerfile.build
+++ b/Dockerfile.build
@@ -1,13 +1,9 @@
-FROM ubuntu:20.04
+FROM golang:1.14-buster
 
-RUN apt-get update && apt-get install -y \
-    golang \
-    libbtrfs-dev \
-    git-core \
-    libdevmapper-dev \
-    libgpgme11-dev \
-    go-md2man \
-    libglib2.0-dev
+RUN apt-get update && \
+    apt-get install -y \
+        libdevmapper-dev \
+        libgpgme11-dev
 
 ENV GOPATH=/
 WORKDIR /src/github.com/containers/skopeo

